### PR TITLE
Initial instructions (from 2020 repo)

### PR DESCRIPTION
**Pull request by loganthomas**
Copied over instructions from last year ([here](https://github.com/enthought/Numpy-Tutorial-SciPyConf-2020))

A few concerns:
- Are we keeping the same slides or do I need to build new ones? I anticipate needing to build new ones since we likely have new material from last year.
- I wasn't sure about the dependencies and `environment.yml` file.
  - I updated the package versions based on the most recent `python-class` environment we used for teaching.
    ```
    - numpy 1.15+ -> numpy 1.17+
    - matplotlib 2.2+ -> matplotlib 3.3+
    - ipython -> ipython 7.16+
    - pyqt 5.7+ -> pyqt5 5.14+
    - (not previously mentioned) jupyter 1.0+
    ```
  - I included a few additional dependencies in the `environment.yml` file:
     ```
    name: numpy-tutorial
    dependencies:
        - numpy 
        - matplotlib
        - pillow
        - jupyter
        - pyqt
        - ipython
    ```
  - What's the point of the `environment.yml` file? Should we tell students to create from this instead of a `conda create -n`? (i.e. `conda env create -f environment.yml`)
  - ~~Is it `pyqt` or `pyqt5`?~~

### CODE REVIEW

**Comment by loganthomas** [comment]
@dbolack We don't need CI checks for this repo.

**Comment by loganthomas** [comment]
`pyqt` not `pyqt5` (see [here](https://github.com/enthought/Numpy-Tutorial-SciPyConf-2020/pull/2/commits/e74ab3c8800123e40b98c54ef01b925f5851a252))

File: README.md [hunk 1]
```diff
@@ -1,2 +1,63 @@
...
+- numpy 1.17+
+- matplotlib 3.3+
+- ipython 7.16+ (for running, experimenting and doing exercises)
+- pyqt 5.14+
+- jupyter 1.0+
+
+If you are using Anaconda, you can create an environment with the necessary packages with:
+
+```
+$ conda create -n numpy-tutorial numpy matplotlib ipython pyqt
```
Comment (achabotl) [inline README.md]:
```suggestion
$ conda create -n numpy-tutorial numpy matplotlib ipython pyqt jupyter pillow
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,63 @@
...
+
+To be able to run the examples, demos and exercises, you must have the following packages installed:
+
+- numpy 1.17+
+- matplotlib 3.3+
+- ipython 7.16+ (for running, experimenting and doing exercises)
+- pyqt 5.14+
+- jupyter 1.0+
+
+If you are using Anaconda, you can create an environment with the necessary packages with:
```
Comment (achabotl) [inline README.md]: Specify using the "Anaconda Prompt app (Windows)" or Terminal.app (macOS).

File: README.md [hunk 3]
```diff
@@ -1,2 +1,63 @@
...
+
+```
+$ conda create -n numpy-tutorial numpy matplotlib ipython pyqt
+```
+
+To test your installation, please execute the `check_env.py` script in the environment where you have installed the requirements.  For example, if you created an Anaconda environment using the instructions above, open up a terminal (or command prompt), navigate to where you have this GitHub repository, and type:
```
Comment (achabotl) [inline README.md]: Like above, let's be specific about the Anaconda Prompt and Terminal apps.

**Review by achabotl — APPROVED** [review]
- Yes, new slides
- `environment.yml` is required by Binder. I wasn't sure if a fully-qualified `environment.yml` force a certain platform so I specified the dependencies by name only. Feel free to adjust. 
- The advantage of using a list of packages to create the env is that it's path independent. But, as you can see in my inline comments, it's easy for them to get out of sync...

File: README.md (hunk 2 above)
Comment (loganthomas) [reply README.md]: https://docs.anaconda.com/anaconda/install/verify-install/#conda

File: README.md (hunk 3 above)
Comment (loganthomas) [reply README.md]: https://docs.anaconda.com/anaconda/install/verify-install/#conda

**Pull request merged**